---
title: 'Heritability and quantitative trait loci of composition and structural characteristics in sorghum grain'

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here
# and it will be replaced with their full name and linked to their profile.
authors:
  - Nikhil Y. Patil
  - admin
  - Robert R. Klein
  - Hector S. Martinez
  - Raul S. Martinez
  - Raul Rodriguez-Herrera
  - William L. Rooney
  - Patricia E. Klein
# Author notes (optional)
#author_notes:
date: '2019-11-05'
doi: 'https://doi.org/10.1080/15427528.2018.1536006'

# Schedule page publish date (NOT publication's date).
#publishDate: '2017-01-01T00:00:00Z'

# Publication type.
# Accepts a single type but formatted as a YAML list (for Hugo requirements).
# Enter a publication type from the CSL standard.
publication_types: ['Research Paper']

# Publication name and optional abbreviated publication name.
publication: In *Journal of Crop Improvement*
#publication_short: In *F*

abstract: Breeding efforts in cereal crops directed toward developing or improving end-use products of grain require assessment of existing phenotypic variance and an understanding of the genetic control of grain quality traits. To this end, a grain sorghum [Sorghum bicolor (L.) Moench] mapping population consisting of 113 F2:7 recombinant inbred lines (RILs) derived from a cross between Sure√±o and RTx430 was evaluated in multiple environments for grain composition (fat, fiber, protein, starch) using near-infrared reflectance spectroscopy (NIRS), and size estimates of grain parts (embryo, vitreous endosperm, floury endosperm, kernel area) using an image-based phenotyping software system. Estimates of broad-sense heritability of grain compositional traits ranged from 0.11 to 0.90, whereas those of grain size ranged from 0.16 to 0.72. Composite interval mapping (CIM) was applied to a single nucleotide polymorphism (SNP)-based linkage map to identify marker-trait associations, and through these efforts, a total of 37 quantitative trait loci (QTL) for grain quality were identified across environments. Each QTL explained between 7 and 23% of the phenotypic variation for a given grain trait. Three of the five QTL that colocalized were for traits with significant negative correlation, which included grain protein content that was negatively correlated with grain starch content. In addition, several traits that were positively correlated (e.g. fat and fiber content) also revealed colocalized QTL. Finally, we compared the present study with previous studies identifying grain composition trait loci in an effort to identify genomic regions controlling grain traits across a diversity of environments and sorghum genotypes.

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin #condimentum.

tags:
  - Plant Breeding
  - Genetics

# Display this page in the Featured widget?
featured: false

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: ''
#url_project: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
image:
  caption: A peanut field in Lubbock, TX
  focal_point: 
  preview_only: false
  orientation: 'none'  # or you can use '0' to indicate no rotation

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
  - example

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: example
---
